README.md

```diff
@@ -41,9 +41,27 @@ The CAPITALIZED words throughout these guidelines have a special meaning:
 
 Refer to [RFC2119](https://www.ietf.org/rfc/rfc2119) for details.
 
+### Enforcing API Guidelines over your OpenAPI Specification
```

Comment:
Enforcing -> is not a great word. I'd suggest something like "Validating your OpenAPI Specification with the adidas API guidelines"